logger/logger.go

```diff
@@ -37,28 +36,26 @@ import (
 )
 
 type Logger struct {
-	ctx            context.Context
-	log            *logrus.Logger
+	Log            *logrus.Logger
 	mu             sync.Mutex
 	lastLogCall    int64
 	debugOverride  bool
 	categoryFilter *regexp.Regexp
 }
 
-func NullLogger() *logrus.Logger {
+// NewNullLogger will create a logger where log lines will
+// be discarded and not logged anywhere.
+func NewNullLogger() *Logger {
```

Comment:
Do we still need this after [this change](https://github.com/grafana/xk6-browser/pull/310/commits)? We can simply pass a nil logger (or not even pass one). If we follow this path, we should also remove other `NewNullLogger` calls around the codebase as well.